nook-desktop.desktop

```diff
@@ -0,0 +1,6 @@
+[Desktop Entry]
+Name=Nook
+Exec=
```

Comment:
exec is empty

Reply:
Actually looking back I'm pretty sure I misread it, I will fill it in